src/App.js

```diff
@@ -1,24 +1,37 @@
 import logo from './logo.svg';
 import './App.css';
+import styled from 'styled-components';
+
+const ButtonGroup = styled.div`
+  display: flex;
+`
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Neutron Technologies</p>
-        <a
-          className="Neutron-Tek"
-          href="https://github.com/neutrontechnologies"
-          target="_blank"
-          rel="noopener noreferrer"
-          title="Check out our organization!"
-        >
-          <button>Organization page</button>
-        </a>
-        
-        
+        <p>Neutron Technologies</p>
+        <ButtonGroup>
+          <a
+            className="Neutron-Tek"
+            href="https://github.com/neutrontechnologies"
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Check out our organization!"
+          >
+            <button>Organization page</button>
```

Comment:
The React component for buttons is <Button>, this is the <button> html tag which doesn't have the react component features. (I know it was like that before, that's our issue).